src/main/java/org/spongepowered/api/world/extent/worker/BlockVolumeWorker.java

```diff
@@ -53,21 +54,21 @@
     /**
      * Applies a mapping operation to all the blocks in the volume and saves the
      * results to the destination volume.
-     *
-     * @param mapper The mapping operation
+     *  @param mapper The mapping operation
      * @param destination The destination volume
+     * @param cause
      */
-    void map(BlockVolumeMapper mapper, MutableBlockVolume destination);
+    void map(BlockVolumeMapper mapper, MutableBlockVolume destination, Cause cause);
```

Comment:
@DDoS I had to change some of these methods to use the `Cause` as I'm planning on implementation to use a customary phase to enter for all the block changes until the worker has completed applying the changes. I'm not 100% sure that I did it all right but, please do comment to clear it up for me.